types/nested-error-stacks/nested-error-stacks-tests.ts

```diff
@@ -0,0 +1,6 @@
+import NestedErrorStacks from 'nested-error-stacks';
```

Comment:
You should use `import name = require('package');` instead.

If you still want to use the syntax `import name from 'package';` you need to use `esModuleInterop` on the compiler settings from your consuming code.